### ISSUE TITLE
Change api url config file option

### ISSUE THREAD

**Issue by esc** [ImmobilienScout24/afp-cli#37]
Does anyone have any issues changing the `api_url` config item to `api-url`?

**Comment by schlomo** [comment]
I would like to have an option to only set the hostname part of the URL - leaving the protocol and path unchanged. ATM I need to set the complete URL even if I only want to change the hostname of the server.

**Comment by esc** [comment]
those are separate issues, mind you.

**Comment by schlomo** [comment]
Yes? My issue is you want me to change my config and I - the end user - voice my opinion about what kind of change I actually need :-)

So, maybe instead of changing the api_url key you add a new key called api-host (or something like this) that solves my problem and probably the problem of all the other people who ever had to change the api_url key :-)

**Comment by schlomo** [comment]
See also #39, though I would love to have a corresponding config item as well.

**Comment by esc** [comment]
@schlomo how do we deal with this? Would propose to change this, when we change to v2

**Comment by esc** [comment]
But maybe we can deprecate it already?

**Comment by schlomo** [comment]
Sure, why not add a deprecation warning for the old form? In 1-2 months we can then remove it.

**Comment by schlomo** [comment]
And advice people to switch to the new `server` variable which will probably obsolete the need for people to change the `api-url` variable altogether.

**Comment by esc** [comment]
Yes, but only if it is found in config, right? I don't want a warning if I am overriding things on the command-line.

**Comment by schlomo** [comment]
If you use the old command line argument then you should be warned. Otherwise not.
